### PR TITLE
Created new Endpoint Update prize

### PR DESCRIPTION
**Pull request by MiguelTrujillo26**
New endpoint Update prize

### CODE REVIEW

File: app/src/db/prize.js [hunk 1]
```diff
@@ -69,6 +69,31 @@ const updateLevel = async (prizeId, level) => {
...
+}
+
+const isRelatedToInventory = async (prizeId) => {
+    const res = await pool.pool.query(`SELECT EXISTS (
+                                                        SELECT 1
+                                                        FROM prize p
+                                                        JOIN company c ON p.company_id = c.company_id
+                                                        JOIN experience e ON c.company_id = e.company_id
+                                                        JOIN inventory i ON e.experience_id = i.experience_id
+                                                        CROSS JOIN jsonb_array_elements(i.prize_pieces) AS pp
```
Comment (andymotagl) [inline app/src/db/prize.js:89]: Let's suppose you have 2M players and 1K prizes per inventory, how long this cross join would take? 
It is not needed to be responded, I would prefer an easier query, something like any inventory created for the given experience

Comment (MiguelTrujillo26) [reply app/src/db/prize.js]: In that case, the new proposal is better because it is straightforward, or could it be better?

Comment (fernando-arevalo-galilei) [reply app/src/db/prize.js]: hey @andymotagl 

how an experience is related when editing prize data?

File: app/src/service/prize.js [hunk 2]
```diff
@@ -51,6 +51,31 @@ const updateLevel = async (prizeId, level) => {
...
+    if (!body.image) throw new BadRequest('Invalid image');
+}
+
+const isRelatedToInventory = async (prizeId) => {
+    const isRelated = await prizeDAO.isRelatedToInventory(prizeId);
+    if(isRelated){
+        throw new BadRequest('Unable to update because the prize is associated with at least one inventory.');
+    }
+    return isRelated;    
+}
```
Comment (fernando-arevalo-galilei) [inline app/src/service/prize.js:77]: this should return only true or false, should not? Then who uses it can decide what to do with the response

**Review by fernando-arevalo-galilei — APPROVED** [review]

**Pull request merged**